ext/opentelemetry-ext-flask/src/opentelemetry/ext/flask/__init__.py

```diff
@@ -29,9 +29,10 @@
 
 .. code-block:: python
 
-    from opentelemetry.ext.flask import FlaskInstrumentor
-    FlaskInstrumentor().instrument()  # This needs to be executed before importing Flask
     from flask import Flask
+    from opentelemetry.ext.flask import FlaskInstrumentor
+
+    Flask = FlaskInstrumentor().instrument(flask_class=Flask)
```

Comment:
Any naming conflicts in this example due to `Flask` imported library and instantiated variable?

Reply:
That's actually the purpose of it, to overwrite the imported `Flask` to make it instrumented.